var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -48,16 +49,21 @@ class Adios(AutotoolsPackage):
     variant('fortran', default=False,
             description='Enable Fortran bindings support')
 
-    variant('mpi', default=True, description='Enable MPI support')
+    variant('serial', default=False, description='Build serial version by fully disabling MPI support')
```

Comment:
There are 124 packages in Spack with an `+mpi` variant and exactly 0 with a `+serial` variant. I think it would be better to call this `mpi`.